.github/workflows/danger-prs.yml

```diff
@@ -2,17 +2,40 @@ name: iOS Consistency Checks
 on:
   pull_request:
     # The restrictions on the types of pull_request event that can trigger this
-    # workflow is intentional. We only want to run these sanity checks when the
```

Comment:
More inclusive language. Thanks again @jkmassel for pointing this out.

Reply:
Nice catch! 👍